inst/base/databox_cpp.h

```diff
@@ -19,17 +19,19 @@
 #ifndef DATABOX_CPP_H
 #define DATABOX_CPP_H
 
-void databox::mevent(double time, int evid) {
-  mrgsolve::evdata ev(time,evid);
+void databox::mevent(double time_, int evid_) {
```

Comment:
Just renaming the `evid` value coming in as an argument here; same consideration as we had for `time` and `time_`.